presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/security/TableAccessControlRule.java

```diff
@@ -54,6 +54,15 @@ public TableAccessControlRule(
         return Optional.empty();
     }
 
+    public Optional<Set<TablePrivilege>> matchSchema(String user, String schemaName)
```

Comment:
I don't like it returns `TablePrivilege`. Table privilates are related to particular table, not entire schema. Maybe return `boolean`?